### DISCUSSION TITLE
People card translation

### DISCUSSION THREAD

**Discussion by coupalm** [microsoft-search/pnp-modern-search#1956] [Q&A]
Hi, how can I translate the search result v4 people popup card in other languages (only in english)? 

![image](https://user-images.githubusercontent.com/11309224/159790215-eae95ded-afd0-4b94-a3da-c638dd96be0f.png)

**Comment by wobba** [comment]
Duplicate of https://github.com/microsoft-search/pnp-modern-search/issues/1321
